src/sys/windows.rs

```diff
@@ -735,6 +736,52 @@ pub(crate) fn from_in6_addr(addr: in6_addr) -> Ipv6Addr {
     Ipv6Addr::from(*unsafe { addr.u.Byte() })
 }
 
+/// This type is not yet in winapi.
+#[repr(C)]
+#[allow(non_camel_case_types)]
+struct sockaddr_un {
```

Comment:
Note to self: pr to add this in winapi: https://github.com/retep998/winapi-rs/pull/968.

Reply:
windows-sys has this definition so I've removed the private one.